commands/list/clusters/command.go

```diff
@@ -32,9 +32,22 @@ var (
 		Use:     "clusters",
 		Aliases: []string{"cluster"},
 		Short:   "List clusters",
-		Long:    `Prints a list of all clusters you have access to.`,
-		PreRun:  printValidation,
-		Run:     printResult,
+		Long: `Prints a list of all clusters you have access to.
+
+Examples:
+
+  gsctl list clusters
+
+  gsctl list clusters --output json
+
+  gsctl list clusters --show-deleting
+
+  gsctl list clusters --selector environment=testing
+
+  gsctl list clusters --sort organization
```

Comment:
```suggestion
  gsctl list clusters --sort org
```

Let's hunch users to save some typing :)